{{ cookiecutter.repo_name }}/README.md

```diff
@@ -9,48 +9,50 @@ Project Organization
     ├── LICENSE
```

Comment:
As with previous comment: Not sure if necessary, but this structure is missing descriptions for:

- `.gitignore`
- `test_environment.py`

Reply:
Descriptions added.